src/arrayMethodSort.js

```diff
@@ -4,9 +4,42 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function(compareFunction = compareStrings) {
     // write code here
+    let isSorting;
+
+    do {
+      isSorting = false;
+
+      for (let i = 0; i < this.length - 1; i++) {
+        const a = this[i];
+        const b = this[i + 1];
+
+        if (compareFunction(a, b) > 0) {
+          this[i] = b;
+          this[i + 1] = a;
+          isSorting = true;
+        }
+      }
+    } while (isSorting === true);
```

Comment:
Keep in mind that you do this as well:
```suggestion
    } while (isSorting);
```